Clarifying how to use groups with OIDC (#485)
minio/docs: source/security/openid-external-identity-management/external-authentication-with-openid-identity-provider.rst

MinIO provides :ref:`built-in policies <minio-policy-built-in>` for basic access
control. You can create new policies using the :mc:`mc admin policy` command, or
by using the MinIO Console. 

> MinIO does not support assigning an identity managed by :abbr:`OIDC (OpenID Connect)` to a MinIO :ref:`group <minio-groups>`. 

Reviewer: @donatello to confirm, but until multi-IDP support is ready IIRC you cannot even create groups in MinIO. as written (previously and now) might imply that you could still create a group in the first place.

I'd classify this as a nit as the behavior overall is still "true".

> Instead, an :abbr:`OIDC (OpenID Connect)` administrator can use the user-specified claim to list multiple MinIO :ref:`policies <minio-policy>` to assign to the user.

Reviewer: ```suggestion
Instead, an :abbr:`OIDC (OpenID Connect)` administrator can use the configured OIDC claim to list multiple comma-separated MinIO :ref:`policies <minio-policy>` to assign to the user (e.g. ``'policy[,policy]'``)
```

Author: 👍

> In this way, the OIDC administrator can create a type of "group" assignment managed entirely within :abbr:`OIDC (OpenID Connect)`.

Reviewer: Style point: "In this way" doesn't add anything. We can cut to the point and just say:

```suggestion
The OIDC administrator can use this method to mimic "group" assignment managed entirely within :abbr:`OIDC (OpenID Connect)`.
```

Author: 👍
